perk-store-services/src/main/resources/locale/addon/PerkStore_az.properties

```diff
@@ -237,7 +238,7 @@ exoplatform.perkstore.button.refresh=Refresh
 exoplatform.perkstore.button.search=Search
 exoplatform.perkstore.button.download=Download
 exoplatform.perkstore.button.loadMore=Load more
-exoplatform.perkstore.button.editProduct=Edit product
+exoplatform.perkstore.button.editProduct=Edit Product
```

Comment:
changing keys will not be impacted and will be overwritten on next Crowdin synchro, please read :
http://developer.exoplatform.org/#id-how-to-update-an-existing-translation-string

Reply:
fixed in crowdin !!